coriolis/tests/test_cache.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+from coriolis import exception
+from coriolis import cache
+from coriolis.tests import test_base
+
+
+class TestGetCacheDecoratorTestCase(test_base.CoriolisBaseTestCase):
```

Comment:
NIT: Class name should simply be `CacheTestCase`, as we should add future methods to test in this class (not just `get_cache_decorator` specific). Please rename it

Reply:
Done, thanks a lot for everythink! :)